akka-actor-tests/src/test/scala/akka/io/dns/internal/AsyncDnsResolverSpec.scala

```diff
@@ -89,6 +89,20 @@ class AsyncDnsResolverSpec extends AkkaSpec(
       }
     }
 
+    "fails if all dns clients fail" in {
+      val dnsClient1 = TestProbe()
+      val dnsClient2 = TestProbe()
+      val r = resolver(List(dnsClient1.ref, dnsClient2.ref))
+      r ! Resolve("cats.com", Ip(ipv4 = true, ipv6 = false))
+      dnsClient1.expectMsg(Question4(1, "cats.com"))
+      dnsClient1.reply(Failure(new RuntimeException("Fail")))
+      dnsClient2.expectMsg(Question4(2, "cats.com"))
+      dnsClient2.reply(Failure(new RuntimeException("Yet another fail")))
+      expectMsgPF(remainingOrDefault) {
+        case Failure(ResolveFailedException(_)) ⇒
```

Comment:
Should we check the message here?

Reply:
I think there is no much value checking it - `ResolveFailedException` is created with only one message.